mysql-test/suite/innodb/t/percona_sys_tablespace_encrypt.test

```diff
@@ -58,6 +59,35 @@ CREATE TABLE t10(a TEXT) TABLESPACE = innodb_system, ENCRYPTION='Y';
 --error ER_ILLEGAL_HA_CREATE_OPTION
 CREATE TABLE t11(a TEXT) TABLESPACE = innodb_system, ENCRYPTION='N';
 
+SET GLOBAL innodb_encrypt_tables=@save_innodb_encrypt_tables;
+
+########################### Additional Test Scenarios ################################
+# 1. Move table from un-encrypted sys tablespace to another encrypted tablespace     #
+# 2. Move table from un-encrypted sys tablespace to another un-encrypted tablespace  #
+######################################################################################
+
+--echo # 1. Move table from un-encrypted sys tablespace to another encrypted tablespace
+CREATE TABLE t1 (a int) tablespace=innodb_system;
```

Comment:
Here & everywhere, please use consistent case
```suggestion
CREATE TABLE t1 (a int) TABLESPACE=INNODB_SYSTEM;
```

Reply:
Fixed. Made inline with the original test
CREATE TABLE t1 (a int) TABLESPACE = innodb_system;